apps/sms/views/conversation/js/conversation.js

```diff
@@ -763,15 +767,14 @@ var ConversationView = {
           }
 
           this.recipients.add(recipient);
-        }.bind(this));
-      }, this);
+
+          this.draft.isEdited = false;
+        });
+      });
 
       // Render draft contents into the composer input area.
       Compose.fromDraft(this.draft);
 
-      // Discard this draft object and update the backing store
-      Drafts.delete(this.draft).store();
```

Comment:
The initial logic for threadless drafts was that they were only really used outside of the new-message panel. So here once it's loaded we can delete it. We were always saving a new one when leaving the panel or on visibilitychange event.
As you can see there was quite a different logic for threadless and thread-bound drafts too.

Now I think we need some new logic to update a draft, keeping its id. Also I wonder if we should delete new message drafts after sending the message -- if (in the future) a user can bookmark a URL, he could bookmark a composer with this draftId, and so maybe he could want to send it several times ? Not for now of course, but a use case to keep in mind.
For example, because we don't have the "reject call then send message" feature, we could imagine that a user saves a draft "I am in a meeting" to be easily recalled.